client/src/Chat.jsx

```diff
@@ -87,21 +92,21 @@ const Messages = ({ user }) => {
 };
 const Chat = () => {
   const [state, stateSet] = React.useState({
-    user: "Olasumbo",
-    content: "",
+    user: process.env.defaultUsername,
+    content: ''
   });
 
   const [postMessage] = useMutation(POST_MESSAGE);
 
   const onSend = () => {
     if (state.content.length > 0) {
       postMessage({
```

Comment:
Dont you see this error when try to post message? I do and the message is not getting sent because of this :(

<img width="695" alt="Screenshot 2022-03-14 at 22 23 17" src="https://user-images.githubusercontent.com/89577597/158254966-fd3052ba-8906-4742-ae9d-dc28c3655b1c.png">
.